Makefile

```diff
@@ -1,6 +1,10 @@
 
 BIN_NAME="terraform-provider-instaclustr"
-VERSION=v1.9.1
+
+VERSION=v1.9.2
+release_version:
```

Comment:
what's this target for?
also it doesn't represent a target, should make it .PHONY

Reply:
Not actually part of this ticket, but it lets us get the version from the makefile for the jenkins tagging. What's a .PHONY do?

Reply:
Makefile targets represent a file to determine "cached builds" - if you `make stuff` and `stuff` file exists then make won't run that target.
Phony says that the target is not a real target (it doesn't create a file with `release_version`), meaning that it will run the target regardless of the presence of "cached file". release_version being a real target will prevent it to run (echo wont happen) if `release_version` file exists. of course that's not going to happen and it's not really a concern, but we should follow the standard